### DISCUSSION TITLE
Stack 란?

### DISCUSSION THREAD

**Discussion by starshb** [starshb/pharmacy_Find#47] [Q&A] [answered]
Stack 란?

**Comment by tjdud22** [comment, marked as answer]
스택은 데이터를 일시적으로 저장하기 위해 사용하는 자료구조입니다.
데이터의 입력과 출력 순서는 후입선출(LIFO:Last In First Out)입니다.

push : 데이터를 넣는 작업
pop : 데이터를 꺼내는 작업
top : push, pop 하는 위치
botoom : stack의 가장 밑바닥 부분
